src/material-experimental/mdc-core/option/_option-theme.scss

```diff
@@ -0,0 +1,68 @@
+@import '@material/list/mixins.import';
+@import '@material/list/variables.import';
+@import '@material/theme/functions.import';
+@import '../../mdc-helpers/mdc-helpers';
+@import '../../../material/core/theming/check-duplicate-styles';
+
+@mixin mat-mdc-option-color($config-or-theme) {
+  $config: mat-get-color-config($config-or-theme);
+
+  @include mat-using-mdc-theme($config) {
+    .mat-mdc-option {
+      // Since this will usually we rendered in an overlay,
+      // we have explicitly set the default color.
+      @include mdc-theme-prop(color, text-primary-on-background);
+      @include mdc-list-item-disabled-text-color($mdc-list-text-disabled-color,
+        $query: $mat-theme-styles-query);
+
+      &:hover:not(.mdc-list-item--disabled),
+      &:focus:not(.mdc-list-item--disabled),
+      &.mat-mdc-option-active,
+
+      // In multiple mode there is a checkbox to show that the option is selected.
+      &.mdc-list-item--selected:not(.mat-mdc-option-multiple):not(.mdc-list-item--disabled) {
+        $color: $mdc-theme-on-surface;
+        background: rgba($color, mdc-states-opacity($color, hover));
+      }
+    }
+
+    .mat-primary .mat-mdc-option.mdc-list-item--selected:not(.mdc-list-item--disabled) {
+      @include mdc-list-item-primary-text-ink-color(primary, $query: $mat-theme-styles-query);
+    }
+
+    .mat-accent .mat-mdc-option.mdc-list-item--selected:not(.mdc-list-item--disabled) {
+      @include mdc-list-item-primary-text-ink-color(secondary, $query: $mat-theme-styles-query);
+    }
+
+    .mat-warn .mat-mdc-option.mdc-list-item--selected:not(.mdc-list-item--disabled) {
+      @include mdc-list-item-primary-text-ink-color(error, $query: $mat-theme-styles-query);
+    }
+  }
+}
+
+@mixin mat-mdc-option-typography($config-or-theme) {
+  $config: mat-get-typography-config($config-or-theme);
+}
+
+@mixin _mat-option-density($config-or-theme) {
```

Comment:
Similarly to my comment above, this should be `mat-mdc-option-density`. By the way: Should this already include density or are you looking into that in a follow-up?

I remember that MDC has density for list-items, but the question might be how this should work in options. e.g. in focused tasks (like menu), density should _not_ apply. So maybe that would need some thinking.

Reply:
I decided not to add density for now since I was going for feature parity. Since the mixin is there, we can easily add it later.

Reply:
Yeah, I was just checking. Can we still rename the mixin here though? Thanks!